testRandom.js

```diff
@@ -6,16 +6,27 @@ var mqtt    = require('./')
   , start   = Date.now() / 1000
   , time
   , errors  = 0
-
+  , randomPacket
 
 function doParse () {
   var parser  = mqtt.parser()
   parser.on('error', onError)
-  parser.parse(crypto.randomBytes(Math.floor(Math.random() * 10)))
+  randomPacket = crypto.randomBytes(Math.floor(Math.random() * 10))
```

Comment:
I also thought we possibly should shape the stochastic distritbution of the first byte so that most packets at least enter the parser and aren't rejected by the first gate (packet type selector)